### PR TITLE
[testing] tree: promote changes from testing-devel at 53098661a83b3c6b26c334540aaec72d3f9cbfc9

### PR DESCRIPTION
**Pull request by coreosbot-releng**
Triggered by @gursewak1997 in https://github.com/coreos/fedora-coreos-streams/issues/458.

### CODE REVIEW

File: kola-denylist.yaml [hunk 1]
```diff
@@ -19,42 +19,50 @@
...
+  streams:
+    - rawhide
+- pattern: ext.config.toolbox
+  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1103
+  snooze: 2022-03-15
+  streams:
+    - rawhide
+- pattern: multipath.day1
+  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1105
+  snooze: 2022-03-07
```
Comment (dustymabe) [inline kola-denylist.yaml]: should we make another commit on top here that makes this not a snooze but a full denial? That way if it happens in two weeks when we spin stable it won't be an issue.

**Review by dustymabe — DISMISSED** [review]
One suggestion otherwise LGTM

File: kola-denylist.yaml (hunk 1 above)
Comment (jlebon) [reply kola-denylist.yaml]: Makes sense to me. (Doesn't this apply to all the other 03-07 ones too?)

Comment (dustymabe) [reply kola-denylist.yaml]: i think all of the other ones are for `branched`/`rawhide`

**Review by dustymabe — APPROVED** [review]
LGTM

**Review by gursewak1997 — APPROVED** [review]
